site/docs/downloads/index.md

```diff
@@ -27,13 +27,13 @@ The main Nessie server serves the Nessie repository using the Iceberg REST API a
 Docker images are multiplatform images for amd64, arm64, ppc64le, s390x.
 
 === "GitHub Container Registry"
-    [GitHub Container Registry](https://ghcr.io/projectnessie/nessie)
+    [![ghcr.io GitHub Container Registry](https://img.shields.io/maven-central/v/org.projectnessie.nessie/nessie?label=ghcr.io+Docker&logo=docker&color=3f6ec6&style=for-the-badge&logoColor=white)](https://github.com/projectnessie/nessie/pkgs/container/nessie)
```

Comment:
Better remove the badge for consistency

Reply:
Why? I precisely added the badge wherever it was missing.

Reply:
See #8934 from yesterday